pyspecProcScripts/load_data.py

```diff
@@ -364,8 +384,13 @@ def proc_spincore_IR(s, fl=None):
     s.setaxis("ph1", r_[0, 2.0] / 4)
     s.setaxis("ph2", r_[0, 2.0] / 4)
     s.reorder(["ph1", "ph2"]).set_units("t2", "s")
+    s.set_units("t2", "s")
+    s.set_prop("coherence_pathway", {"ph1": 0, "ph2": -1})
```

Comment:
again, where is this coming from? doesn't this have an echo?

Reply:
edited according to discussion [here](https://jmfrancklab.slack.com/archives/CTD29CZPE/p1719521574408469)